main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/yagreut/onboardingAssignment/models"
+	"github.com/yagreut/onboardingAssignment/repository"
+	"github.com/yagreut/onboardingAssignment/service"
+
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		logrus.Fatal("Usage: go run main.go <input_file.json>")
+	}
+
+	inputFile := os.Args[1]
```

Comment:
Let's enhance the task - add a secret scanning capability:
For each file, except for the large files, search whether it contains a GitHub Token.
Explore how to do it :)
If something is not clear, please ask